### PR TITLE
added VMs dropdown in top consumers card

### PR DESCRIPTION
**Pull request by afreen23**
[pull/481](https://github.com/kubevirt/web-ui-components/pull/481)

### CODE REVIEW

File: frontend/public/storage/components/storage-overview/storage-overview.jsx [hunk 1]
```diff
@@ -38,6 +39,10 @@ const CONSUMERS_SLCLASSES_REQUESTED_CAPACITY_QUERY = '(sort(topk(5, sum(avg_over
 const CONSUMERS_SLCLASSES_USED_CAPACITY_QUERY = '(sort(topk(5, sum(avg_over_time(kubelet_volume_stats_used_bytes[1h]) * on (namespace,persistentvolumeclaim) group_left(storageclass) kube_persistentvolumeclaim_info) by (storageclass))))[10m:1m]';
 const CONSUMERS_PODS_REQUESTED_CAPACITY_QUERY = '(sort(topk(5, sum(avg_over_time(kube_persistentvolumeclaim_resource_requests_storage_bytes[1h]) * on (namespace,persistentvolumeclaim) group_left(pod) kube_pod_spec_volumes_persistentvolumeclaims_info) by (pod))))[10m:1m]';
 const CONSUMERS_PODS_USED_CAPACITY_QUERY = '(sort(topk(5, sum(avg_over_time(kubelet_volume_stats_used_bytes[1h]) * on (namespace,persistentvolumeclaim) group_left(pod) kube_pod_spec_volumes_persistentvolumeclaims_info) by (pod))))[10m:1m]';
+
+const CONSUMERS_VMS_REQUESTED_CAPACITY_QUERY = '';
```
Comment (suomiy) [inline frontend/public/storage/components/storage-overview/storage-overview.jsx]: what are the empty queries for?

File: frontend/public/storage/components/storage-overview/storage-overview.jsx [hunk 2]
```diff
@@ -77,6 +84,9 @@ const resourceMap = {
   pvcs: {
     resource: getResource(PersistentVolumeClaimModel),
   },
+  vms: {
```
Comment (suomiy) [inline frontend/public/storage/components/storage-overview/storage-overview.jsx]: how are the vms used in the storage-overview?

File: frontend/public/storage/components/storage-overview/storage-overview.jsx (hunk 1 above)
Comment (afreen23) [reply frontend/public/storage/components/storage-overview/storage-overview.jsx]: for the VMs dropdown in top consumers card

Comment (suomiy) [reply frontend/public/storage/components/storage-overview/storage-overview.jsx]: why are they empty?

Comment (afreen23) [reply frontend/public/storage/components/storage-overview/storage-overview.jsx]: the query is not ready till yet..

Comment (suomiy) [reply frontend/public/storage/components/storage-overview/storage-overview.jsx]: ok, can you mark this PR WIP then?

Comment (afreen23) [reply frontend/public/storage/components/storage-overview/storage-overview.jsx]: is it ok if the query can be sent with other updated queries in a followup PR ?

Comment (suomiy) [reply frontend/public/storage/components/storage-overview/storage-overview.jsx]: It is not much useful without it. IMO it is better to wait for the query.

Comment (afreen23) [reply frontend/public/storage/components/storage-overview/storage-overview.jsx]: alright :+1:

**Comment by afreen23** [comment]
@mareklibra @rawagner @suomiy .... Can you guys help on providing the query for VMs...??

**Comment by suomiy** [comment]
Sorry don't really know + no prometheus atm. 

The query should be only about the vm disks, right?
Another approach (just for the requested storage) could be to parse the requested capacity from vm definitions or from the pvcs.

**Comment by cloudbehl** [comment]
@suomiy The storage requested capacity can be found from the kubelet metrics but the what exactly we are looking at is the labels that define this storage is provided to VM. So we can calculate on basis of that.

@afreen23 @rawagner @mareklibra ^^

**Pull request merged**